storage/cql_iface/cql_iface.py

```diff
@@ -0,0 +1,153 @@
+from typing import Tuple, List
+from uuid import UUID
+
+from storage.cql_iface.tests.mockIStorage import IStorage
+from .config import _hecuba2cassandra_typemap
+from .cql_comm import CqlCOMM
+from ..storage_iface import StorageIface
+
+"""
+Mockup on how the Cassandra implementation of the interface could work.
+"""
+
+
+class CQLIface(StorageIface):
+    hcache_datamodel = []
+    # DataModelID - DataModelDef
+    data_models_cache = {}
+    # StorageID - DataModelID
+    object_to_data_model = {}
+    # Object Name - Cache
+    hcache_by_name = {}
+    # Object's class - Cache
+    hcache_by_class = {}
+    # StorageID - Cache
+    hcache_by_id = {}
+
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def _check_values_from_definition(definition):
```

Comment:
You should look at the levels in this method; as more nested levels, more difficult to read and easier to spot bugs.
if `definition.__origin__` is not Tuple, this code will work and not check the type (there is no else). Is this expected?

The same code could look something like:
```python

value_iter = None
if isinstance(definition, dict):
  value_iter = definiton.values()
elif isinstance(definition, (list, set, tuple)):
  value_iter = iter(definition)


if value_iter is not None:
   for v in value_iter:
     CQLIface._check_values_from_definition(v)  
else:
   try:
     my_type = definition.__origin__
   except AttributeErrror:
     my_type = definition

   if my_type not in _hecuba2cassandra_typemap:
       raise TypeError(f"The type {definition} is not supported")
```